dataset/pascal_voc.py

```diff
@@ -24,18 +24,22 @@ class PascalVoc(Imdb):
     is_train : boolean
         if true, will load annotations
     """
-    def __init__(self, image_set, year, devkit_path, shuffle=False, is_train=False,
-            names='pascal_voc.names'):
+    def __init__(self, image_set, year, devkit_path, shuffle=False, is_train=False, class_names=None,
+            names='pascal_voc.names', true_negative_images=False):
         super(PascalVoc, self).__init__('voc_' + year + '_' + image_set)
         self.image_set = image_set
         self.year = year
         self.devkit_path = devkit_path
         self.data_path = os.path.join(devkit_path, 'VOC' + year)
         self.extension = '.jpg'
         self.is_train = is_train
+        self.true_negative_images = true_negative_images
 
-        self.classes = self._load_class_names(names,
-            os.path.join(os.path.dirname(__file__), 'names'))
+        if class_names is not None:
+            self.classes = class_names.replace(' ', '').split(',')
```

Comment:
class_names.strip().split(',') will be better IMO.

Reply:
I'll change that